frontend/lib/parsing/parsing-queries.cpp

```diff
@@ -411,9 +416,38 @@ const Location& locateId(Context* context, ID id) {
 // this is just a convenient wrapper around locating with the id
 const Location& locateAst(Context* context, const AstNode* ast) {
   CHPL_ASSERT(!ast->isComment() && "cant locate comment like this");
+  CHPL_ASSERT(ast); // TODO: Will crash instead of return nullptr...
```

Comment:
Pointless assert because the previous line (`ast->isComment()`) would segfault before this assertion would fail. Consider swapping / moving to first line of `locateAst`.

Also, I'm not making much sense of the comment.

Reply:
Ideally, the query wouldn't crash?